tasks/reduce-snapshot/tests/test-reduce-snapshot-no-namespace-parameter.yaml

```diff
@@ -0,0 +1,97 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-reduce-snapshot-no-namespace-parameter
+spec:
+  description: |
+    Run the reduce task to reduce to a single component without the namespace parameter
```

Comment:
can you use the actual parameter name? Took me a few mins to figure this one out

Reply:
fixed description